layerfetcher/layers_test.go

```diff
@@ -352,9 +352,14 @@ func TestPollLayerBlocks_FetchLayerBlocksError(t *testing.T) {
 			}
 			return map[types.Hash32]chan fetch.HashDataPromiseResult{types.RandomHash(): ch}
 		}).Times(numPeers)
+	tl.mLayerDB.EXPECT().SaveHareConsensusOutput(gomock.Any(), layerID, gomock.Any()).DoAndReturn(
+		func(_ context.Context, _ types.LayerID, blockID types.BlockID) interface{} {
+			assert.NotEqual(t, blockID, types.EmptyBlockID)
+			return nil
+		}).Times(1)
 
 	res := <-tl.PollLayerContent(context.TODO(), layerID)
-	assert.Equal(t, ErrLayerDataNotFetched, res.Err)
```

Comment:
can you change the test name to `FetchLayerBlocksErrorIgnored`? thanks

Reply:
done